testsuite/module-publish/src/main.rs

```diff
@@ -148,7 +148,7 @@ use std::collections::HashMap;"#,
 fn write_package(file: &mut File, package_path: PathBuf, package_name: &str) -> String {
     println!("Building package {}", package_name);
     // build package
-    let package = BuiltPackage::build(package_path, BuildOptions::default())
+    let package = BuiltPackage::build(package_path, BuildOptions::move_2())
```

Comment:
Can we specify optimizer options as well? Not sure if those are enabled by default, cc @vineethk

Reply:
There are a set of optimizations enabled by default.

Reply:
Great!